app/controllers/simplified-account/settings/worldpay-details/worldpay-details.controller.test.js

```diff
@@ -0,0 +1,62 @@
+const ControllerTestBuilder = require('@test/test-helpers/simplified-account/controllers/ControllerTestBuilder.class')
+const sinon = require('sinon')
+const { expect } = require('chai')
+const Service = require('@models/Service.class')
+const GatewayAccount = require('@models/GatewayAccount.class')
+
+const mockResponse = sinon.spy()
+
+const ACCOUNT_TYPE = 'live'
+const SERVICE_ID = 'service-id-123abc'
+
+const { req, res, call } = new ControllerTestBuilder('@controllers/simplified-account/settings/worldpay-details/worldpay-details.controller')
+  .withService(new Service({
+    external_id: SERVICE_ID
+  }))
+  .withAccountType(ACCOUNT_TYPE)
+  .withAccount(new GatewayAccount({
```

Comment:
this is a good idea, we should probably update the ControllerTestBuilder to create a new GatewayAccount when calling `withAccount` instead of relying on individual tests